app/src/main/java/fnsb/macstransit/MapsActivity.java

```diff
@@ -2,39 +2,35 @@
 
 import android.util.Log;
 import android.view.Menu;
-import android.widget.Toast;
 
 import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Circle;
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.MapStyleOptions;
 import com.google.android.gms.maps.model.Marker;
-import com.google.android.gms.maps.model.MarkerOptions;
 
 import java.util.ArrayList;
```

Comment:
Personal goal: Eliminate the use of ArrayLists in future use, as it requires extraneous overhead that may impact ram performance.

Reply:
Since this is mainly used for intercompatability between methods, perhaps outsourcing the methods would achieve this goal along with trimming down the overall file size 🤔